Packs/StarterPack/pack_metadata.json

```
@@ -0,0 +1,14 @@
+{
+    "name": "Starter Pack",
+    "description": "Starter Pack for learning how to build new content in Cortex XSOAR",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Your Company Name",
```

@ShahafBenYakir 
Let's put this as Cortex XSOAR so it is clear that this pack is from us.
```suggestion
    "author": "Cortex XSOAR",
```